pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -38,16 +38,16 @@ module.exports = {
     I.click(agentLinkLocator);
     I.waitForElement(this.fields.pmmAgentLocator, 60);
     I.waitForElement(this.fields.inventoryTable, 60);
+    I.scrollPageToBottom();
     const numberOfServices = await I.grabNumberOfVisibleElements(
       "//span[contains(text(), '" +
         serviceId +
         "')]/following-sibling::span[contains(text(),'status: RUNNING')]"
     );
     if (
-      service_name === 'rds-mysql56' ||
```

Comment:
why remove rds from here?

Reply:
i've done that because I already map 'mysql' substring, but i'll add includes('rds') just in case